addons/crm/static/src/js/tours/crm.js

```diff
@@ -1,100 +1,113 @@
+odoo.define('crm.crm_steps', function (require) {
+"use strict";
+
+var core = require('web.core');
+
+var CrmAdditionalTourSteps = core.Class.extend({
+
+    _get_crm_iap_lead_steps: function () {
+        return [];
+    },
+
+});
+
+return CrmAdditionalTourSteps;
+
+});
```

Comment:
Can you make a regular function instead of a whole class?

var CrmAdditionalTourSteps = function () {...}
No need for a whole class if you're just defining a function.

Side note: it's always camelCase in JS, never snake_case.